### PR TITLE
support dotenv

### PR DESCRIPTION
**Pull request by murugaratham**
It would be easier and more developer friendly to use .env file to configure that long list of environment variables yea?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -59,3 +59,4 @@ typings/
 
 # next.js build output
 .next
+.vscode/settings.json
```
Comment (LoneRifle) [inline .gitignore]: nitpick - add a few newlines and a descriptive comment (`# VS Code settings` would suffice)

File: index.js [hunk 2]
```diff
@@ -48,4 +49,4 @@ app.listen(
   err => err
     ? console.error('Unable to start MockPass', err)
     : console.warn(`MockPass listening on ${PORT}`)
-)
+)
```
Comment (LoneRifle) [inline index.js]: nitpick - could you restore the newline here please?

File: index.js [hunk 3]
```diff
@@ -2,6 +2,7 @@
 const fs = require('fs')
 const express = require('express')
 const path = require('path')
+require('dotenv').config();
```
Comment (LoneRifle) [inline index.js]: nitpick - trailing `;` - if you could make the change adhere better to the lib's eslint rules that would be great

File: .gitignore (hunk 1 above)
Comment (murugaratham) [reply .gitignore]: done

**Comment by LoneRifle** [comment]
@murugaratham thanks for the contribution! I'll try to release this over the next few days

**Pull request merged**